service/environment.go

```diff
@@ -0,0 +1,121 @@
+package service
+
+import (
+	"context"
+
+	"release-manager/pkg/apierrors"
+	"release-manager/pkg/dberrors"
+	"release-manager/service/model"
+
+	"github.com/google/uuid"
+)
+
+type EnvironmentService struct {
+	authSvc    model.AuthService
+	projectSvc model.ProjectService
+	repository model.EnvironmentRepository
+}
+
+func NewEnvironmentService(authSvc model.AuthService, projectSvc model.ProjectService, repo model.EnvironmentRepository) *EnvironmentService {
```

Comment:
I think it's an unnecessary overhead to have environment creation as a separate service. It's part of the project resource, doesn't have a seaparate resource endpoint, hence I would put it under the project service. Not that you don't have to have a service per resource either, rather structure it per domain.